README.md

```diff
@@ -26,8 +32,8 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 
 ----
 
-*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*  
-*Documentation/examples are licensed under [Creative Commons BY-SA 4.0](/docs/LICENSE).*  
+*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*
```

Comment:
The trailing whitespace is intentional due to Gitlab and other markdown implementations using that as a new line.